### ISSUE TITLE
style.css Needed

### ISSUE THREAD

**Issue by Firelight97** [Firelight97/Firelight97.github.io#2]
Website is no good without style!

**Comment by codewithjk** [comment]
hi, are you learning how to use GitHub

**Comment by Firelight97** [comment]
No. I'm learning more of HTML while making a website in the process. I've learned quite a bit of Git and GitHub thanks to a website that I enjoyed before.
